DistributionModels/weibull.py

```diff
@@ -40,6 +44,25 @@ def FitLow(self,data, tailSize, isSorted=False, gpu=0):
         return to_return
 
 
+    def FitLowReversed(self,data, tailSize, isSorted=False, gpu=0):
```

Comment:
This function seems to simply set `self.reversed` flag, should this be absorbed in the existing `FitLow` function rather than a new function?

Reply:
I'm confused isn't reversed FitLow essentially FitHight